### PR TITLE
add new rule best Framework Rule

### PR DESCRIPTION
**Pull request by tomek-i**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️ Missing rule

> 2. What was changed?

✏️This pull request adds the addition of a new rule.
The new rule provides information on the best frameworks to build an Admin Interface for web applications, including React/Next.js, Blazor, and Angular. These updates and additions aim to improve the functionality and usability of the application.

> 3. Did you do pair or mob programming (list names)?

✏️ Solo

### CODE REVIEW

File: rules/do-you-know-the-best-framework-to-build-an-admin-interface/rule.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+authors:
+  - title: Tom Iwainski
+    url: https://www.ssw.com.au/people/thomas-iwainski
+related:
+  - use-nextjs
+  - why-react-is-great
+  - why-blazor-is-great
+  - why-angular-is-great
+created: 2024-10-224T00:00:00.001Z
+guid: abcdefgh-1234-a1b2-xyza-ffffffffffff
```
Comment (wicksipedia) [inline rules/do-you-know-the-best-framework-to-build-an-admin-interface/rule.md:15]:
```suggestion
guid: be0d9e5f-8580-4997-9abd-d7130ec9b240
```

File: rules/do-you-know-the-best-framework-to-build-an-admin-interface/rule.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+seoDescription: Do you know the best Framework to build an Admin Interface for your Web App?
```
Comment (wicksipedia) [inline rules/do-you-know-the-best-framework-to-build-an-admin-interface/rule.md:2]:
```suggestion
seoDescription: Find the best framework for your admin interface. Explore top choices like React, Blazor, and Angular, based on your tech stack and project needs
```

**Review by wicksipedia — APPROVED** [review]
test pass with suggestions

**Comment by tiagov8** [comment]
Hi @tomek-i 

I've made some content design changes to your new rule.

**Review by tiagov8 — CHANGES_REQUESTED** [review]
- [ ] 1. Please add to category

**Comment by tomek-i** [comment]
> Hi @tomek-i
> 
> I've made some content design changes to your new rule.
> 
> And I noticed it's missing belonging to a category... so I added a task in my review
> 
> PS. There is another unrelated change in this PR (on the email placeholders UPPERCASING). This part looks fine, but it seems like the same as this other PR #9473 . Unclear how this happened, but check so it can be avoided next time ;)

Yes, I did see that too. The first change i made with the UI and then wrote a new rule but that resulted in the UI to throw errors and people suggested to do it via github. I think it branched off from that other PR that why it included both changes.

Will fix the category and figure out the validation error sometime today. Thanks

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**